services/blockstorage/test/block_sync_test.go

```diff
@@ -211,8 +211,11 @@ func TestSyncCompletePetitionerSyncFlow(t *testing.T) {
 }
 
 func TestSyncNeverStartsWhenBlocksAreCommitted(t *testing.T) {
+	// this test may still be flaky, it runs commits in a busy wait loop that should take longer than the timeout,
```

Comment:
how can you make it completely deterministic?

Reply:
raise the timeouts - if we go into 100 and 50ms  scenario the probability of it being flaky is low enough (we see 5-15ms of freezes sometimes, so 50ms should be safe)
but then its 'slow', not sure how to refactor it

Reply:
nope, this is just reducing probability of failure
let's do this - merge it, but open an issue and we can maybe hold a discussion tomorrow at Weekly about it

Reply:
sounds good, will do